docs/excel/custom-functions-runtime.md

```diff
@@ -0,0 +1,149 @@
+---
+ms.date: 09/18/2018
+description: Excel custom functions use a new JavaScript runtime, which differs from the standard Add-ins WebView control runtime.
+title: Runtime for Excel Custom Function Add-ins
+---
+
+# Runtime for Excel custom function add-ins
+
+Custom functions extend Excel’s capabilities using a new JavaScript runtime. This runtime utilizes a sandboxed JavaScript engine rather than a web browser. Additionally, it prioritizes performance, allowing you to run thousands of custom functions simultaneously.  
```

Comment:
is "prioritizes" the right word here?

Reply:
So the JS runtime is weird because it performs better than the standard WebView for this type of mass calculation, but I want to avoid making a direct comparison between the two. Maybe: "Because custom functions do not need to render UI elements, their runtime is optimized for performing calculations instead; you can run thousands of custom functions simultaneously."